app/api/dao/admin.py

```diff
@@ -60,6 +69,15 @@ def revoke_admin_user(user_id, data):
 
         new_admin_user = UserModel.find_by_id(admin_user_id)
 
+        #For test purposes
```

Comment:
You can remove this comment, because this is not only for tests, but actually "production" code that was missing ;)